localhost-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
missed something - what is this for?

Reply:
This is because in order for the SW to run on localhost, you have to enable HTTPS in development. It's a pain but I put some info in the README about it. You don't need it generally because I disable the SW in development by default... but if you do want to work on the SW for example then you need to run `npm run example:sw` which will use these keys for HTTPS